diff --git a/CMakeLists.txt b/CMakeLists.txt
index ca9b04a..231ec7a 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,7 @@ add_definitions(-D_VERSION="v${MAJOR_VERSION}.${MINOR_VERSION}.${PATCH_VERSION}"
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/cmake/Modules")
 
 # In debug mode check code style issues
 if(CMAKE_BUILD_TYPE MATCHES Debug)
@@ -74,7 +74,7 @@ set(CMAKE_INSTALL_PREFIX "/usr")
 install(FILES ${PROJECT_SOURCE_DIR}/installation/touchegg.conf DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/touchegg) # /usr/share/touchegg/touchegg.conf
 install(FILES ${PROJECT_SOURCE_DIR}/installation/touchegg.service DESTINATION /lib/systemd/system)
 install(FILES ${PROJECT_SOURCE_DIR}/installation/touchegg.desktop DESTINATION /etc/xdg/autostart)
-install(PROGRAMS ${CMAKE_BINARY_DIR}/touchegg DESTINATION ${CMAKE_INSTALL_BINDIR}) # /usr/bin/touchegg
+install(PROGRAMS ${PROJECT_BINARY_DIR}/touchegg DESTINATION ${CMAKE_INSTALL_BINDIR}) # /usr/bin/touchegg
 
 set(CPACK_PACKAGE_NAME touchegg)
 set(CPACK_PACKAGE_VERSION_MAJOR "${MAJOR_VERSION}")
diff --git a/cmake/Modules/FindCairo.cmake b/cmake/Modules/FindCairo.cmake
index a5f3c40..59b808e 100644
--- a/cmake/Modules/FindCairo.cmake
+++ b/cmake/Modules/FindCairo.cmake
@@ -2,4 +2,4 @@ find_package(PkgConfig)
 pkg_check_modules(CAIRO REQUIRED cairo)
 
 include(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(CAIRO DEFAULT_MSG CAIRO_FOUND)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(Cairo DEFAULT_MSG CAIRO_FOUND)
diff --git a/cmake/Modules/FindGLib.cmake b/cmake/Modules/FindGLib.cmake
index cbcd41d..c08d0df 100644
--- a/cmake/Modules/FindGLib.cmake
+++ b/cmake/Modules/FindGLib.cmake
@@ -2,4 +2,4 @@ find_package(PkgConfig)
 pkg_check_modules(GLIB REQUIRED glib-2.0)
 
 include(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(GLIB DEFAULT_MSG GLIB_FOUND)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(GLib DEFAULT_MSG GLIB_FOUND)
diff --git a/cmake/Modules/FindGtk3.cmake b/cmake/Modules/FindGtk3.cmake
index 896e539..7a1a756 100644
--- a/cmake/Modules/FindGtk3.cmake
+++ b/cmake/Modules/FindGtk3.cmake
@@ -2,4 +2,4 @@ find_package(PkgConfig)
 pkg_check_modules(GTK3 REQUIRED gtk+-3.0)
 
 include(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(GTK3 DEFAULT_MSG GTK3_FOUND)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(Gtk3 DEFAULT_MSG GTK3_FOUND)
diff --git a/cmake/Modules/FindLibinput.cmake b/cmake/Modules/FindLibinput.cmake
index 3315de5..3d65ba0 100644
--- a/cmake/Modules/FindLibinput.cmake
+++ b/cmake/Modules/FindLibinput.cmake
@@ -2,4 +2,4 @@ find_package(PkgConfig)
 pkg_check_modules(LIBINPUT REQUIRED libinput)
 
 include(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(LIBINPUT DEFAULT_MSG LIBINPUT_FOUND)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(Libinput DEFAULT_MSG LIBINPUT_FOUND)
diff --git a/cmake/Modules/FindLibudev.cmake b/cmake/Modules/FindLibudev.cmake
index 6f8decc..27af6f1 100644
--- a/cmake/Modules/FindLibudev.cmake
+++ b/cmake/Modules/FindLibudev.cmake
@@ -2,4 +2,4 @@ find_package(PkgConfig)
 pkg_check_modules(LIBUDEV REQUIRED libudev)
 
 include(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(LIBUDEV DEFAULT_MSG LIBUDEV_FOUND)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(Libudev DEFAULT_MSG LIBUDEV_FOUND)
diff --git a/cmake/Modules/FindPugiXML.cmake b/cmake/Modules/FindPugiXML.cmake
index 35d184b..97aefbd 100644
--- a/cmake/Modules/FindPugiXML.cmake
+++ b/cmake/Modules/FindPugiXML.cmake
@@ -2,4 +2,4 @@ find_package(PkgConfig)
 pkg_check_modules(PUGIXML REQUIRED pugixml)
 
 include(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(PUGIXML DEFAULT_MSG PUGIXML_FOUND)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(PugiXML DEFAULT_MSG PUGIXML_FOUND)
diff --git a/cmake/Modules/FindXInput.cmake b/cmake/Modules/FindXInput.cmake
index 99310f5..66e0182 100644
--- a/cmake/Modules/FindXInput.cmake
+++ b/cmake/Modules/FindXInput.cmake
@@ -2,4 +2,4 @@ find_package(PkgConfig)
 pkg_check_modules(XINPUT REQUIRED xi)
 
 include(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(XINPUT DEFAULT_MSG XINPUT_FOUND)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(XInput DEFAULT_MSG XINPUT_FOUND)
diff --git a/cmake/Modules/FindXTest.cmake b/cmake/Modules/FindXTest.cmake
index e39b749..edb0636 100644
--- a/cmake/Modules/FindXTest.cmake
+++ b/cmake/Modules/FindXTest.cmake
@@ -2,4 +2,4 @@ find_package(PkgConfig)
 pkg_check_modules(XTEST REQUIRED xtst)
 
 include(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(XTEST DEFAULT_MSG XTEST_FOUND)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(XTest DEFAULT_MSG XTEST_FOUND)
diff --git a/cmake/Modules/FindXrandr.cmake b/cmake/Modules/FindXrandr.cmake
index 452c442..ee34c3f 100644
--- a/cmake/Modules/FindXrandr.cmake
+++ b/cmake/Modules/FindXrandr.cmake
@@ -2,4 +2,4 @@ find_package(PkgConfig)
 pkg_check_modules(XRANDR REQUIRED xrandr)
 
 include(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(XRANDR DEFAULT_MSG XRANDR_FOUND)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(Xrandr DEFAULT_MSG XRANDR_FOUND)
